src/clojure/nrepl/middleware/interruptible_eval.clj

```diff
@@ -97,7 +99,9 @@
                                 eof)))]
                 (when-not (identical? input eof)
                   (try
-                    (let [value (eval-fn input)]
+                    (let [value (if eval-fn
```

Comment:
What's the reason for this `if`? That's not clear to me.

Reply:
Added a comment to explain.